specification.markdown

```diff
@@ -1155,6 +1155,32 @@ The value represents data that **MAY** be stored within the EBML Element.
 
 The `value` attribute is **REQUIRED**.
 
+#### minver
+
+Within an EBML Schema, the XPath of the `@value` attribute is
+`/EBMLSchema/element/restriction/enum/@minver`.
+
+The `minver` (minimum version) attribute stores a nonnegative integer that
+represents the first version of the docType to support the enum value. The value
+**MUST** be greater than or equal to the `minver` attribute value of the EBML Element it belongs to.
+
+The `minver` attribute is **OPTIONAL**. If the `minver`
+attribute is not present, then the enum value has the same value as the `minver` attribute of the
+EBML Element it belongs to.
+
+#### maxver
+
+Within an EBML Schema, the XPath of the `@value` attribute is
+`/EBMLSchema/element/restriction/enum/@maxver`.
+
+The `maxver` (maximum version) attribute stores a nonnegative integer that
+represents the last or most recent version of the docType to support
+the enum value. `maxver` **MUST** be greater than or equal to the `minver` attribute.
```

Comment:
Doesn't this imply that for the `maxver` attribute to exist, the `minver` attribute must exist as well? Isn't the following more what you'd want to express:

* Enum's `maxver` must not be less than enum's `minver` if that exists
* Enum's `maxver` must not be less than element's `minver` if that exists

Reply:
This text is mostly the same as the `maxver` of the `<element>`, which is also not very precise. The idea is that it's greater than or equal to the enum minimum version, whether it is written or assumed.